pidtool.py

```diff
@@ -112,43 +136,61 @@ def create_resamplers(options):
+        pid_variables += [#transformed ProbNN with log( var/(1-var) ) => more stable distribution
+                          '{}_V3ProbNNK_Trafo', '{}_V3ProbNNpi_Trafo', '{}_V3ProbNNmu_Trafo', '{}_V3ProbNNp_Trafo', '{}_V3ProbNNe_Trafo', '{}_V3ProbNNghost_Trafo',
+                          '{}_V2ProbNNK_Trafo', '{}_V2ProbNNpi_Trafo', '{}_V2ProbNNmu_Trafo', '{}_V2ProbNNp_Trafo', '{}_V2ProbNNe_Trafo', '{}_V2ProbNNghost_Trafo'
+                          ]
     kin_variables = ['{}_P', '{}_Eta','nTracks']
 
 
-    with open('raw_data.json') as f:
+    with open(options.config) as f:
         locations = json.load(f)
     if options.particles:
         locations = [sample for sample in locations if sample["particle"] in options.particles]
     if options.both_magnet_orientations:
         locations = [sample for sample in locations if sample["magnet"]=="Up"] # we use both maagnet orientations on the first run
     for sample in locations:
         binning_P = rooBinning_to_list(GetBinScheme(sample['branch_particle'], "P", options.binning_name)) #last argument takes name of user-defined binning
-        binning_ETA = rooBinning_to_list(GetBinScheme(sample['branch_particle'], "ETA", options.binning_name)) #last argument takes name of user-defined binning 
+        binning_ETA = rooBinning_to_list(GetBinScheme(sample['branch_particle'], "ETA", options.binning_name)) #last argument takes name of user-defined binning
         binning_nTracks = rooBinning_to_list(GetBinScheme(sample['branch_particle'], "nTracks", options.binning_name)) #last argument takes name of user-defined binning
-    	if options.both_magnet_orientations:
-            if sample["magnet"]=="Up":  
+        if options.both_magnet_orientations:
+            if sample["magnet"]=="Up":
                 data =  [options.location + '/{particle}_Stripping{stripping}_MagnetUp.root'  .format(**sample)]
                 data += [options.location + '/{particle}_Stripping{stripping}_MagnetDown.root'.format(**sample)]
-                resampler_location = '{particle}_Stripping{stripping}_MagnetAny.pkl'.format(**sample)
+                resampler_location = options.saveto + '/{particle}_Stripping{stripping}_MagnetAny.pkl'.format(**sample)
         else:
             data = [options.location + '/{particle}_Stripping{stripping}_Magnet{magnet}.root'.format(**sample)]
-            resampler_location = '{particle}_Stripping{stripping}_Magnet{magnet}.pkl'.format(**sample)
+            resampler_location = options.saveto + '/{particle}_Stripping{stripping}_Magnet{magnet}.pkl'.format(**sample)
         if os.path.exists(resampler_location):
             os.remove(resampler_location)
         resamplers = dict()
         deps = map(lambda x: x.format(sample['branch_particle']), kin_variables)
         pids = map(lambda x: x.format(sample['branch_particle']), pid_variables)
         for pid in pids:
             if "DLL" in pid:
-                target_binning = np.linspace(-150, 150, 300) # binning for DLL
```

Comment:
I don't think this should be hard-coded here.

Instead we should make it easier for the user to pass a user-defined binning that doesn't come from RooBinning.

Maybe we should remove the whole RooBinning import/conversion stuff from the module and put it in its own file into a `tools` subdir.